src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -69,7 +69,7 @@ fi
 
 if [ "$EXECUTION_DIR" != "$BASEDIR" ]; then
   echo "Copying files into execution dir."
-  ln -f $BASEDIR/* $EXECUTION_DIR || exit -1
+  ln -f $BASEDIR/* $EXECUTION_DIR || echo "Link failed, fallback to copy"; cp -run $BASEDIR/* $EXECUTION_DIR || echo "Still errors, attempting to continue"
```

Comment:
Chatted offline and we are going to try to eliminate this copying into the Execution directory all together.